src/tools/annotation/AngleTool.js

```diff
@@ -274,7 +275,10 @@ export default class AngleTool extends BaseAnnotationTool {
       const str = '00B0'; // Degrees symbol
 
       return (
-        data.rAngle.toString() + String.fromCharCode(parseInt(str, 16)) + suffix
+        toGermanNumberStringTemp(data.rAngle) +
```

Comment:
We don't use the AngleTool, only the cobb angle. Or is this here just for the sake of consistency?

Reply:
You are right - I was thinking about whether or not to implement it everywhere or only where needed. I then decided to go for consistency. But it definitely was not needed.

Reply:
Makes sense and it's fine, the old way is in the comments